src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -82,7 +82,8 @@ public void SetGet_Roundtrips()
         [InlineData(true)]
         public void ProxySetViaEnvironmentVariable_DefaultProxyCredentialsUsed(bool useProxy)
         {
-            bool envVarsSupported = UseManagedHandler || !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            // TODO: Issue #25640 It should be enabled for managed Handler on all platforms
+            bool envVarsSupported = !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
             if (!envVarsSupported)
```

Comment:
Instead of doing this, you could add an ActiveIssue attribute like ```[ActiveIssue(25640, TestPlatforms.Windows)]``` which will skip the test only on Windows.

Reply:
Yes, this is the preferred way. And it makes it easier to find "disabled" tests in the repo.